code/modules/urist/structures/closets.dm

```diff
@@ -245,3 +245,27 @@
 		/obj/item/storage/firstaid/rad,
 		/obj/item/storage/firstaid/toxin,
 		/obj/item/storage/med_pouch/radiation = 2)
+
+/singleton/closet_appearance/urist/sarcophagus
+	decals = null
+	extra_decals = null
+	base_icon = 'icons/urist/obj/sarcophagus.dmi' //uses the icon_states base, open, interior, lock / light / sparks (if possible), welded
```

Comment:
This .dmi file is missing the following icon_states:
`lock`, `light`,`interior`,`welded`,`sparks`